docs/user-guide/configuring.md

```diff
@@ -775,6 +775,22 @@ You can also use your `.gitignore` file:
 
 Any file that follows the standard ignore file format can be used. Keep in mind that specifying `--ignore-path` means that any existing `.eslintignore` file will not be used. Note that globbing rules in `.eslintignore` follow those of `.gitignore`.
 
+### Using eslintIgnore in package.json
+
+If an .eslintIgnore file is not found and an alternate file is not specified, eslint will look in package.json for an eslintIgnore key to check for files to ignore.
```

Comment:
Nitpick: The file is `.eslintignore`, all lowercase. Also, I think it would be good for both `.eslintignore` and `eslintIgnore` to have backticks around them to help call them out visually. Thanks!